### ISSUE TITLE
Destructuring object yield a "Parsing error: Unexpected token .."

### ISSUE THREAD

**Issue by D219** [eslint/eslint#9568]
Windows 10 Enterprise
Node 8.7.0
npm 5.4.2
eslint 4.9.0

No parser

<details>
<summary>Configuration</summary>

```js
{
	"env": {
		"es6": true,
		"node": true
	},
	"extends": "eslint:recommended",
	"rules": {
		"indent": [
			"error",
			"tab"
		],
		"linebreak-style": [
			"error",
			"unix"
		],
		"quotes": [
			"error",
			"single"
		],
		"semi": [
			"error",
			"always"
		],
		"max-len": ["warn"]
	}
}
```
</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

Code:

```js
function split(o) {
	let {a, b, ...n} = o; // [eslint] Parsing error: Unexpected token ... (2, 16)
	return {a, b, n};
}

let s = {a: 1, b: 2, c: {d: 4, e: 5}, f: 6};

console.log(s);
console.log(split(s));
```

```bash
eslint ./test.js
```

Also happen in VS Code (with eslint plug in)

**What did you expect to happen?**
Since the code work as expected, no error to be raised.

**What actually happened? Please include the actual, raw output from ESLint.**
ESLint output a error on the destructuring.

```bash
PS C:\Users\Jeremy Rigaud\Documents\GIT\BrainWeb.module.Melomind> eslint.cmd .\test.js

C:\Users\D219\Documents\GIT\test.js
  2:13  error  Parsing error: Unexpected token ..

✖ 1 problem (1 error, 0 warnings)
```

**Comment by platinumazure** [comment]
Object spread is experimental syntax and so is not supported by default in the espree (default) parser.

You can enable it using this configuration:

```js
{
    "parserOptions": {
        "ecmaFeatures": {
            "experimentalObjectRestSpread": true
        }
    },
    "rules": {} /* etc. */
}
```

**Comment by not-an-aardvark** [comment]
Closing because this isn't an issue with ESLint.
